src/theme/lists.scss

```diff
@@ -12,9 +12,6 @@
   &.top-padding {
     margin-top: 40px;
   }
-  ion-item {
-    padding-left: 19px;
```

Comment:
I suggest leaving this and putting `padding-left: 16px;` for those specific items in the settings css